app/controllers/api/v1/applications_controller.rb

```diff
@@ -16,8 +16,6 @@ def update
         response = rest_application_service.update(application_params.merge(id: params[:id]))
         if response
           render json: true
-        else
```

Comment:
this is handled already by rails, and this way we provide a bit more info to the console.logs on the front end